File: app/src/main/java/cn/com/szw/lib/myframework/utils/RxBus.java
package cn.com.szw.lib.myframework.utils;

import com.hwangjr.rxbus.Bus;

public final class RxBus {
    private static Bus sBus;

    public synchronized static Bus get() {
        if (sBus == null) {
            sBus = new Bus();
        }
        return sBus;
    }
}

File: Flibrary/src/main/java/com/szw/framelibrary/utils/RxBus.kt
package com.szw.framelibrary.utils

import com.hwangjr.rxbus.Bus

object RxBus {
    private var sBus: Bus? = null

    @Synchronized
    fun get(): Bus {
        if (sBus == null) {
            sBus = Bus()
        }
        return sBus as Bus
    }
}